### ISSUE TITLE
implement file input

### ISSUE THREAD

**Issue by oleksandrzelentsov** [oleksandrzelentsov/levenstein_distance#4]
# Cases
## New behavior cases
- giving explicit filename argument (with optional index argument, explained afterwards)
```
$ ./dist --filename <filename>
$ ./dist --filename <filename> --index <index>
```
- having `lwords.txt` file in working directory (with optional index argument, explained afterwards)
```
$ ls
dist lwords.txt
$ ./dist
```
OR
```
$ ./dist --index <index>
```

## Old behavior
Persists regardless of filesystem state.
```
$ ./dist <word1> <word2>
```

# Behavior description
New behavior case should print a list of measurements sorted by the results of measurements.
Example:
```
w3, w2 = 3
w2, w1 = 2
w1, w3 = 1
```
Words are taken from the file. Each word is compared with each another word (order does not matter).
If the `index` argument is given, it should compare all words only to the word in a file, referenced by the value of `index` argument. If there is no word in file under given index, print an error and usage message.

**Comment by oleksandrzelentsov** [comment]
List of words to test out:
- HNIATASA
- HNATIASA
- HNATASIA
- HNIATIASA
- HNIATIASSA
- HNIATASJA
- HNIATYSIA
- HNATIESA
- IHNATIESA
- HNIATIESA
- IHNIATIESA
- HNETIESA
- HNETIESSA
- HNETYSA
- HNYTESA
- HNITESA
- HNITESSA
- HNITEZA
- HNITIASA
- HNITASIA

**Comment by oleksandrzelentsov** [comment]
stuck on reading file, details:
2a51b8059989a77e0244f4096c50c5f31d83b1bf
UPD:
solved

**Comment by oleksandrzelentsov** [comment]
http://cboard.cprogramming.com/c-programming/87932-reading-unicode-file.html
http://www.cplusplus.com/reference/cwchar/

**Comment by oleksandrzelentsov** [comment]
closed by #13

**Comment by oleksandrzelentsov** [comment]
closed by #13
